src/db.rs

```diff
@@ -1716,16 +1717,10 @@ impl DBWithThreadMode<MultiThreaded> {
     }
 
     /// Returns the underlying column family handle
-    pub fn cf_handle(&self, name: &str) -> Option<BoundColumnFamily> {
-        self.cfs
-            .cfs
-            .read()
-            .unwrap()
-            .get(name)
-            .map(|cf| BoundColumnFamily {
-                inner: cf.inner,
-                multi_threaded_cfs: PhantomData,
-            })
+    pub fn cf_handle(&self, name: &str) -> Option<Arc<BoundColumnFamily>> {
```

Comment:
@carllin I think we can avoid the handler approch you mentioned in this way?

1. take `Arc<CF>`
2. call `DB::drop_cf()`
3. pass the `Arc<CF>` to the background cf cleaner thread
4. the thread waits until `Arc::strong_count == 1`.
5. drop!